src/sinks/humio_logs.rs

```diff
@@ -174,6 +180,58 @@ mod integration_tests {
         });
     }
 
+    #[test]
+    fn humio_type() {
+        let mut rt = runtime();
+
+        rt.block_on_std(async move {
```

Comment:
I think [#[tokio::test]](https://docs.rs/tokio/0.2.22/tokio/attr.test.html) can be used instead creating runtime and call `block_on_std`.

Reply:
I actually tried that on Friday after Bruce suggested it in #3175 but it looks like the Humio sink still has a dependency on tokio01 so I ended up getting an error that the sink was shutdown when I tried to send to it. I think the last remaining dependency is through the tls package so it can probably be swapped soon when https://github.com/timberio/vector/pull/3188 is merged.